gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -776,6 +778,7 @@ public BasicInputArea(Composite parent, Settings settings, Theme theme,
       batLabels[1] = createLabel(batGroup, "ms");
       if (caps.getHasPowerRail()) {
         batPowerRail = createCheckbox(batGroup, "Collect Power Rails", sBatt.getCollectPowerRail());
+        //TODO: @nexa to add the checkbox (b/241746026)
```

Comment:
`// TODO(b/241746026): ... `is the normal format for adding TODOs (there's no auto syntax for GitHub but useful for other stuff

Reply:
Got it, thanks